source/seismology/introduction.rst

```diff
@@ -1,2 +1,88 @@
 简介
 ====
+
+:本节贡献者: |姚家园|\（作者）、
+             |田冬冬|\（作者）
+:最近更新日期: 2021-04-23
+:预计阅读时间: 10 分钟
+
+----
+
+地震学是什么？
+--------------
+
+Keiiti Aki 和 Paul G. Richards 编写的经典地震学教材
+《Quantitative Seismology: Theory and Methods》对地震学的定义如下：
```

Comment:
I think this is the correct name:
```suggestion
《Quantitative Seismology》对地震学的定义如下：
```

Reply:
See https://www.ldeo.columbia.edu/~richards/Aki_Richards.html. The 2002 version is actually called **Quantitative Seismology**.

But the definition for seismology in 2002 version changes a little.

> Seismology is the scientific study of mechanical vibrations of the Earth. Quantitative seismology is based on data called seismograms, which are recordings of the vibrations, which in turn may be caused artificially by man-made explosions, or caused naturally by earthquakes and volcanic eruptions.

I prefer the new definition and want to change to it. How do you think?